syft/execution/plan.py

```diff
@@ -306,11 +307,19 @@ def get_args_shape(self):
 
         return [ph.expected_shape for ph in self.role.input_placeholders()]
 
-    def translate_with(
-        self, plan_translator: "sy.execution.translation.abstract.AbstractPlanTranslator"
-    ):
+    def add_translation(self, plan_translator: "AbstractPlanTranslator"):
         return plan_translator(self).translate()
 
+    def remove_translation(self, plan_translator: Union[str, AbstractPlanTranslator] = "default"):
+        if plan_translator == "default":
```

Comment:
Could we make this a real translator so the type is just `AbstractPlanTranslator`?